### ISSUE TITLE
Help with React-Redux connect - Could not find "store"

### ISSUE THREAD

**Issue by invivtas92** [ryandrewjohnson/react-localize-redux#102]
Hi, 

I am having issues when trying to use react-redux connect to connect React component to redux store.

I have looked at the redux helpers examples here https://ryandrewjohnson.github.io/react-localize-redux-docs/#redux-helpers and it looks like it should work with react-localize-redux. 

I used the code sandbox demo (https://codesandbox.io/s/14xp1xy9ql) and added react-redux to the dependencies list and added the following code to Main.js

```
const mapStateToProps = state => {
  currentLanguage: getActiveLanguage(state.localize).code;
};

export default connect(mapStateToProps)(Main);
```
But I received the following error. 

`Could not find "store" in either the context or props of "Connect(Main)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(Main)".`

Anyone knows what I am missing?

**Comment by ryandrewjohnson** [comment]
Are you able to provide a codesandbox example of the error?

**Comment by invivtas92** [comment]
Here is the sandbox link with the error https://codesandbox.io/s/81450lm8k8.

**Comment by invivtas92** [comment]
Yes, I initially thought LocalizeProvider is meant to replace react-redux's Provider. Wrapping the application with Provider and LocalizeProvider as follows seems to solve the issue (assuming this is the proper way of doing it)

```
<LocalizeProvider>
  <Provider store={this.state.store}>
    <Router> 
      <Route path="/" component={props => 
        <Main 
          onToggleClick={this.onToggleReduxClick} 
          toggleValue={this.state.isUsingRedux} 
        />
      } />
    </Router>
  </Provider>
</LocalizeProvider>
```

**Comment by JemarJones** [comment]
@invivtas92  I thought the exact same, its very misleading the way its laid out in the documentation.

**Comment by xanaxx** [comment]
@ryandrewjohnson this thing is still in docs. 

https://ryandrewjohnson.github.io/react-localize-redux-docs/#what-if-i-want-to-use-redux

I think it should be like in invivtas92 comment.

**Comment by ryandrewjohnson** [comment]
@DGulshan please feel free to submit a PR With this update in the docs repo. 
https://github.com/ryandrewjohnson/react-localize-redux-docs

It will get in a lot quicker that way, as I’m super busy right now and won’t have the time to look at this for a while.

**Comment by odykyi** [comment]
just wrap Main.js component to <Provider>

it was helpful for me

**Comment by azharuddinsayed** [comment]
i am using Reactjs.net and getting the error as 
JsScriptException: Script threw an exception.
JavaScriptEngineSwitcher.ChakraCore.JsRt.JsErrorHelpers.ThrowIfError(JsErrorCode errorCode)

JsRuntimeException: Invariant Violation: Could not find "store" in the context of "Connect(Login)". Either wrap the root component in a <Provider>, or pass a custom React context provider to <Provider> and the corresponding React context consumer to Connect(Login) in connect options.

 <Provider store={store}>
still i am getting the error 
please help me .
